releasenotes-builder/src/main/java/com/github/checkstyle/CliProcessor.java

```diff
@@ -296,6 +300,8 @@ private static Options buildOptions() {
             "Access token secret for Twitter.");
         options.addOption(OPTION_TWITTER_PROPERTIES, true,
             "Properties for publication on Twitter.");
+        options.addOption(OPTION_VALIDATE_VERSION, false,
+            "Whether release notes labels should be validated based on release number.");
```

Comment:
It vise versa.
We validate release number to match what we have in notes.
Notes are based on commits, we can not change them.
All we can do and should do is correct release number.

Reply:
Was a bit confused since you mention `Maintainer will make decision to bump version in pom or change labels.` in original issue. Fixed though. Done.

Reply:
You are correct, but usually it results in correction of version number